staging/src/k8s.io/kube-controller-manager/go.mod

```diff
@@ -15,6 +16,7 @@ replace (
 	k8s.io/api => ../api
 	k8s.io/apimachinery => ../apimachinery
 	k8s.io/client-go => ../client-go
+	k8s.io/cloud-provider => ../cloud-provider
```

Comment:
@cheftako did you expect this direction or the reverse? This seems related to https://groups.google.com/forum/?utm_medium=email&utm_source=footer#!msg/kubernetes-sig-api-machinery/hp3uQzAIOsU/njQXHAdvAAAJ ... a sketch of the expected final shape/location for config and controller code might be helpful